<launch>
	<node pkg="tb_vision" type="lk_tracker.py" name="lk_tracker" output="screen">

		<remap from="input_rgb_image" to="/camera/rgb/image_raw" />

		<rosparam>
            gf_maxCorners: 200
            gf_qualityLevel: 0.02
            gf_minDistance: 7
            gf_blockSize: 10
            gf_useHarrisDetector: False
            gf_k: 0.04
            feature_size: 1
            show_text: True
        </rosparam>
        
	</node>
</launch>